src/main/java/chess/domain/Command.java

```diff
@@ -0,0 +1,42 @@
+package chess.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum Command {
+    START("start"),
+    MOVE("move"),
+    END("end");
```

Comment:
```suggestion
    MOVE("move"),
    END("end"),
    ;
```

다음에 추가될 enum을 생각해서 위처럼 쓰기도 합니다 :)

Reply:
감사합니다!